src/backend/storeManagers/storeManagerCommon/games.ts

```diff
@@ -218,6 +218,17 @@ export function getGameProcessName(gameInfo: GameInfo): string | undefined {
   ]?.processName
 }
 
+export function getExecutableAndArgs(executableWithArgs: string): {
+  executable: string
+  launchArgs: string
+} {
+  const match = executableWithArgs.match(/^(.*?\.(exe|app|bin|sh))/i)
+  const executable = match ? match[0] : ''
+  const launchArgs = executableWithArgs.replace(executable, '').trim()
+
+  return { executable, launchArgs }
+}
+
```

Comment:
Can we move this to a separate module and add a unit test for it?

Reply:
done